Makefile

```diff
@@ -1,39 +1,40 @@
-COVERAGE_REPORT := coverage.txt
-COVERAGE_PROFILE := profile.out
-COVERAGE_MODE := atomic
+# Package configuration
+PROJECT = enry
+COMMANDS = cli/enry
+
+# Including ci Makefile
+MAKEFILE = Makefile.main
+CI_REPOSITORY = https://github.com/src-d/ci.git
+CI_FOLDER = .ci
+
+# If you need to build more than one dockerfile, you can do so like this:
+# DOCKERFILES = Dockerfile_filename1:repositoryname1 Dockerfile_filename2:repositoryname2 ...
+
+$(MAKEFILE):
+	@git clone --quiet $(CI_REPOSITORY) $(CI_FOLDER); \
+	cp $(CI_FOLDER)/$(MAKEFILE) .;
```

Comment:
If you just need the Makefile.main file, it may be simpler and more efficient to just get it:

`wget https://raw.githubusercontent.com/src-d/ci/master/Makefile.main`